engine/src/engine.ts

```diff
@@ -192,14 +192,28 @@ export default class Engine {
 
     const move = _move.trim();
     let moveToShow = move;
+    let oldPower: number[] = null;
     if (move.includes(" " + Command.Pass + " ")) {
       moveToShow = move + " returning " + this.player(this.playerToMove).data.tiles.booster;
+    } else if (move.includes(" " + Building.GaiaFormer + " ") && !move.includes("using")) {
+      oldPower = Object.values(this.player(this.playerToMove).data.power);
     }
```

Comment:
If we're going to do multiple checks here it's better to standardize it / use regexps.

Have the regexes declared locally in the file or as static variables of the Engine class.

```ts
const passRegex = new RegExp(`\\b${Command.Pass}\\b(\\S+)?$`);
const gaiaformerRegex =new RegExp(`\\b${Building.GaiaFormer} \\S+$`);
```

Those regexp will check that nothing extra is added, by allowing only one word after the searched item.

It will also match the last pass of a game, when no new booster is picked up.